src/components/approvallist/ApprovalHeader.tsx

```diff
@@ -13,50 +14,65 @@ const StyledFirstRow = styled.div`
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
+  margin-bottom: 16px;
+  padding: 0px 16px;
 `;
 
 export const ApprovalHeader = observer(() => {
   const { uiStore } = useContext(StoreContext);
+  const [approvalDialogOpen, setApprovalDialogOpen] = useState(false);
+  const theme = useTheme();
 
   return (
     <>
+      {approvalDialogOpen && (
+        <div style={{ position: 'absolute', top: '0px', height: '100%', width: '100%' }}>
+          <ApprovalDialog onCancel={() => setApprovalDialogOpen(false)} />
+        </div>
+      )}
       <StyledFirstRow>
-        <Title size="md">Approvals</Title>
-        <Settings />
+        <Typography variant="h4" fontWeight={700}>
+          Approvals
+        </Typography>
+        <Box display="flex" flexDirection="row" gap={2}>
+          <Settings />
+          <Button
+            variant={uiStore.selectedApprovals.length === 0 ? 'outlined' : 'contained'}
+            disabled={uiStore.selectedApprovals.length === 0}
```

Comment:
Nit: you could create a `hasSelectedApprovals` constant or similar to need not compare twice.